README.md

```diff
@@ -105,10 +108,13 @@ func main() {
 	// Produce messages to topic (asynchronously)
 	topic := "myTopic"
 	for _, word := range []string{"Welcome", "to", "the", "Confluent", "Kafka", "Golang", "client"} {
-		p.Produce(&kafka.Message{
+		err = p.Produce(&kafka.Message{
 			TopicPartition: kafka.TopicPartition{Topic: &topic, Partition: kafka.PartitionAny},
 			Value:          []byte(word),
 		}, nil)
+                if err != nil {
+                    fmt.Println("Message could not be enqueued: ", err)
```

Comment:
Use fmt.Printf() to be consistent with existing code.

Reply:
Done!

Reply:
Done